### PR TITLE
Added some mods to your wonderfull Bundle

### PR DESCRIPTION
**Pull request by symio**
Hello,

I have make some enhancements to your Bundle.

I found your Bundle to match my needs but I see that some charts that I was needing were missing.

I added the ability to attach callbacks to charts events.
I added some missing charts.
I added support for javascript Date object.
I have cleaned up your original code a little bit.

I hope you'll like my additions.

Thank you for the original bundle.

Best regards,

Huby Franck

### CODE REVIEW

**Comment by saadtazi** [comment]
Wow! Thank you so much.

I'll do some code review

File: Resources/views/chart/visualizationChart.html.twig [hunk 1]
```diff
@@ -1,7 +1,32 @@
-var data{{ id }} = new google.visualization.DataTable({{ data | json_encode | replace({"\"[new Date]": 'new Date', "[new Date]\"": ""}) | raw }});
+{% if chartType == 'Calendar' or chartType == 'Timeline' %}
+    {% set data = data | json_encode | replace({"\"new Date[[[": 'new Date(', "new Date\"": ""}) | replace({"]]]\"": ')', "]]]\"": ")"}) | replace({"id\":\"role_tooltip": "role\": \"tooltip", "id\":\"role_tooltip": "role\": \"tooltip"}) %}
```
Comment (saadtazi) [inline Resources/views/chart/visualizationChart.html.twig]: Do you think there is a better way to deal with the "date" transformation you are doing here?

Comment (symio) [inline Resources/views/chart/visualizationChart.html.twig]: Yes I'm pretty sure there is a better way but I search for quickest way that do not interact with tooltip or string content (I had issue by generating new Date(2016, 2, 1) and strings ending by ')'.

I plan to enhance this point directly in the Data php classes when I had enough time to make a snippet sufficiency strong to convert directly Datetime php objects in the good Json return and avoid using json_encode and replace in twig.

**Comment by saadtazi** [comment]
Sorry for the delay. I just published v1.2 to packagist.

**Pull request merged**